packages/algob/src/lib/algo-operator.ts

```diff
@@ -344,7 +354,73 @@ export class AlgoOperatorImpl implements AlgoOperator {
       creator: flags.sender.addr,
       txId: txInfo.txId,
       confirmedRound: confirmedTxInfo[confirmedRound],
-      appID: appId
+      appID: appId,
+      timestamp: Math.round(+new Date() / 1000)
+    };
+  }
+
+  /**
+   * Update programs for a contract.
+   * @param sender Account from which call needs to be made
+   * @param payFlags Transaction Flags
+   * @param appId ID of the application being configured or empty if creating
```

Comment:
```suggestion
   * @param appId index of the application being configured or empty if creating
```

Reply:
also, what do we mean by "empty if creating"? Can we update ssc during creation?

Reply:
this must be wrong then, i moved this function here, do you think it is possible ?

Reply:
removed